rules/integrations/cloud_defend/discovery_suspicious_network_tool_launched_inside_a_container.toml

```diff
@@ -0,0 +1,81 @@
+[metadata]
+creation_date = "2023/04/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/04/26"
+
+[rule]
+author = ["Elastic"]
+description = "This rule detects commonly abused network utilities running inside a container. Network utilities like nc, nmap, dig, tcpdump, ngrep, telnet, mitmproxy, zmap can be used for malicious purposes such as network reconnaissance, monitoring or exploitation, and should be monitored closely within a container."
+from = "now-6m"
```

Comment:
Any reason to keep the `from` and `interval` values low? Is it something specific to container TTL by chance. Just curious.

Reply:
I was just sticking with our default values, do you forsee a problem here?